src/components/TransactionBlock/KlimaStakingTransactionBlock.tsx

```diff
@@ -446,14 +454,7 @@ const KlimaStakingTransactionBlock = ({
         showQuickInputButtons
         accountType={selectedAccountType}
       />
-      <NetworkAssetSelectInput
-        label="To"
-        selectedNetwork={supportedChains[1]}
-        selectedAsset={klimaAsset}
-        disabled={true}
-        walletAddress={selectedAccountType === AccountTypes.Contract ? accountAddress : providerAddress}
-        accountType={selectedAccountType}
-      />
+      <NetworkAssetInfoCard label="To" networkDetails={supportedChains[1]} assetDetails={klimaAsset} />
```

Comment:
@parag-chauhan I don't think we need a separate component for this maybe you can pass a property to `<NetworkAssetSelectInput {...args} readOnly />` so this `readOnly` property should handle the things

Reply:
@MayukhSamadder2000 Thanks for the suggestion. Updated to code.